ASDKListKit/ASDKListKitTests/ASListTestObject.m

```diff
@@ -0,0 +1,46 @@
+//
+//  ASListTestObject.m
+//  AsyncDisplayKit
+//
+//  Created by Adlai Holler on 12/25/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#import "ASListTestObject.h"
+
+@implementation ASListTestObject
+
+- (instancetype)initWithKey:(id)key value:(id)value {
+  if (self = [super init]) {
+    _key = [key copy];
+    _value = value;
+  }
+  return self;
+}
+
+- (instancetype)copyWithZone:(NSZone *)zone {
+  return [[ASListTestObject alloc] initWithKey:self.key value:self.value];
+}
+
+
```

Comment:
nit: Extra return here.  Is there a standard on protocol names?  Would it make sense for the pragma mark to say <IGListDiffable> or <IGListDiffable> protocol ?